### PR TITLE
DM-35895: Use band for the dataId not filter

### PR DESCRIPTION
**Pull request by timj**
Gen3 uses band for coadds. If people want to retrieve by
physical filter it would likely need a change in API
or much more cleverness in trying to inspect the dataset type.

### CODE REVIEW

File: doc/lsst.afw.multiband/multiband.rst [hunk 1]
```diff
@@ -480,17 +480,14 @@ to load an exposure from a file:
...
 
     from lsst.afw.image import MultibandExposure
-    from lsst.daf.persistence import Butler
+    from lsst.daf.butler import Butler
 
-    # This is an example dataset on lsstdev which may be out of date,
-    # replace with a local dataset
-    DATA_DIR = "/datasets/hsc/repo/rerun/RC/w_2018_22/DM-14547"
-    butler = Butler(inputs=DATA_DIR)
+    DATA_DIR = "/repo/main"
```
Comment (erykoff) [inline doc/lsst.afw.multiband/multiband.rst:485]: I don't think that `/repo/main` works anywhere ...

**Review by erykoff — APPROVED** [review]
Minor comment on the doc; otherwise if these aren't being used/tested, should we also deprecate them?

File: doc/lsst.afw.multiband/multiband.rst (hunk 1 above)
Comment (timj) [reply doc/lsst.afw.multiband/multiband.rst:485]: It's meant to be an example. Also, K-T was pondering defining `/repo/main` as a butler alias at SLAC.

Comment (timj) [reply doc/lsst.afw.multiband/multiband.rst:485]: Hmm. Nothing is going to work without a collections parameter either here or in the `fromButler` method.

**Pull request merged**